flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -606,6 +614,61 @@ class FlxGamepad implements IFlxDestroyable
 		return getAnalogYAxisValue(Stick);
 	}
 
+	@:allow(flixel.input.gamepad.FlxGamepadManager)
+	private function handleAxisMove(axis:Int, newVal:Float, oldVal:Float)
+	{
+		newVal = applyAxisFlip(newVal, axis);
+		oldVal = applyAxisFlip(oldVal, axis);
+		
+		//check to see if we should send digital inputs as well as analog
+		var stick:FlxGamepadAnalogStick = this.getAnalogStickByAxis(axis);
+		if (stick.mode == ONLY_DIGITAL || stick.mode == BOTH)
+		{
+			var neg = stick.digitalThreshold * -1;
+			var pos = stick.digitalThreshold;
+			var digitalButton = -1;
+			
+			trace("axis(" + axis + ") val = " + newVal);
+			
+			//pressed/released for digital LEFT/UP
+			if (newVal < neg && oldVal >= neg)
```

Comment:
The left/up and right/down blocks seem a little redundant and could probably be abstracted into a helper function?